deploy/chart/values.yaml

```diff
@@ -5,6 +5,8 @@ logLevel: 5
 nameOverride: ""
 fullnameOverride: ""
 
+namespace: ""
```

Comment:
Same here, it should be explained why this value exists. People will get confused as to why `namespace` exists here and how it relates to Helm's `--namespace`.

```suggestion
# This namespace allows you to define where the services are installed into.
# If not set then they use the namespace of the release.
# This is helpful when installing cert manager as a chart dependency (sub chart).
namespace: ""
```